### ISSUE TITLE
Centered whole notes and rests in last bar

### ISSUE THREAD

**Issue by Aurivox** [musescore/MuseScore#20295]
### Your idea

It would be nice if whole tones and whole rests entered with the keyboard were output centered in the final bar, as shown in the example. With large scores this otherwise creates a very unsettling picture

### Problem to be solved

Whole Rests, wenn entered with the keyboard are shown left in bars, instead of been centered like whole rests in empty bars. In final bars whole notes are placed left, instead of been centered like in printed scores 
![Bildschirmfoto 2023-12-03 um 19 15 00](https://github.com/musescore/MuseScore/assets/103463374/b854bd39-dae1-4665-9b1a-07e5a651a613)

**Comment by Aurivox** [comment]
Yes, both is clear to me. But looking only at the final bar with a whole note and and the measure rest I would appreciate if there is a possibility to center the whole note within the bar like in professional scores.

![Bildschirmfoto 2023-12-04 um 21 28 55](https://github.com/musescore/MuseScore/assets/103463374/210408f3-c9b7-4d0c-b609-928740fb8040)

**Comment by MarcSabatella** [comment]
See my comment above.  What you see centered is not a whole rest, it's a measure rest - there is an important different.  A measure rest is *already* present in empty measures by default; you don't have to do a thing to create one.  But you can also enter one at any time with Ctrl+Shift+Delete.

In the future, please use the Support forum on musescore.org when you have questions about how to use MuseScore.

**Comment by oktophonie** [comment]
Marc, this is a feature request to have notes of a duration that fills the whole bar centered in the bar, just like like bar rests are. This is a style commonly seen in older printed music.

**Comment by Jojo-Schmitz** [comment]
So a full-measure chord

**Comment by rpatters1** [comment]
In the Prior Art category, Finale centers whole rests entered into bars as whole-bar rests until another rest is added to it. This may or may not be desirable, but it leads to problems in the musicxml import. One of the most common ways to achieve a fermata over an empty rest in Finale is to enter a "real" whole note and then attach the fermata to it. (It is possible to enter the fermata over an empty bar in Finale, but it can't be done as an articulation.)

The result is that musicxml imported from Finale no longer shows these rests as centered. Perhaps this should be a separate issue aimed at the musicxml import.

**Comment by Jojo-Schmitz** [comment]
Came up again (sort of) in https://musescore.org/en/node/387835, along with a snippet from Elaine Gould's "Behind Bars":
<img width="483" height="271" alt="Image" src="https://github.com/user-attachments/assets/8adbd50a-a00d-4c56-9092-bbc8f29f7252" />
